types/src/main/java/tech/pegasys/artemis/util/uint/UInt64.java

```diff
@@ -320,6 +321,13 @@ public Bytes8 toBytesBigEndian() {
     return Bytes8.wrap(array);
   }
 
+  public Bytes32 toBytes32BigEndian() {
```

Comment:
I would better use `Bytes32.leftPad(UInt64.toBytesBigEndian())` since it explicitly specify how these 8 bytes are padded

Reply:
Thanks 👍

Reply:
I mean using this (`Bytes32.leftPad(UInt64.toBytesBigEndian())`) in code explicitly , since 
- this is very domain specific conversion for including this method to pretty abstract `UInt64`
- the name `toBytes32BigEndian` doesn't reflect semantics (the same ways as `toBytes` name)

Reply:
Yep, agree